metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/BaseJcrProvider.java

```diff
@@ -416,7 +468,7 @@ protected StringBuilder startBaseQuery(String... props) {
         } else {
             boolean start = true;
             for (String prop : props) {
-                if (! start) {
+                if (!start) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2583)